### PR TITLE
feat: Using a single short duration display format

### PR DESCRIPTION
**Pull request by yock**
re: #67

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -15,19 +15,9 @@ def duration_display(minutes)
...
-
-  def long_duration_display(minutes)
-    hours = minutes / 60
-    minutes = minutes - (hours * 60)
-
-    minute_string = minutes > 0 ? "#{minutes} #{'minute'.pluralize(minutes)}" : ""
-    hour_string = hours > 0 ? "#{hours} #{'hour'.pluralize(hours)}" : ""
-
-    "#{hour_string} #{minute_string}".strip
+    "#{hour_string}:#{minute_string}".strip
```
Comment (J3RN) [inline app/helpers/application_helper.rb]: This implementation yield some odd results, based on how the previous one worked. In the previous implementation, I didn't want to have results like "0h 1m" or "1h 0m", so I made it produce "1m" and "1h" instead, respectively. I did this by setting hour_string or minute string to an empty string if the hour or minute was 0, respectively, up on lines 18 and 19.

All that to say, to simplify this a bit, this line should be `"#{'%02d' % hours}:#{'%02d' % minutes}"` and lines 18 and 19 removed.

Comment (yock) [inline app/helpers/application_helper.rb]: Thanks @J3RN. I've added a new PR which formats things with leading zeros. I was able to Postgres (and then the app) running and eyeballed this myself. Now, for instance, a 30 minute duration reads as "00:30" using your specified change. I'd be keen to find a more readable way to write this, but if you're happy, I'm happy.

Comment (J3RN) [inline app/helpers/application_helper.rb]: @yock Thanks for those, this looks great!

I understand your concern about readability, and it's a concern that I share. RuboCop suggests using `hours.format('%02d')` over `'%02d' % hours` (likewise for minutes, of course). Does that seem like a reasonable readability improvement?

Comment (yock) [inline app/helpers/application_helper.rb]: I like it. I'll make the change and push it up.

**Comment by J3RN** [comment]
Fixes #67

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (yock) [inline app/helpers/application_helper.rb]: @J3RN `Fixnum` doesn't contain the `format` method, but it is on `Kernel`. The change I pushed up calls `format('%02d', hours)` instead.

Comment (J3RN) [inline app/helpers/application_helper.rb]: Whoops! Good call. These changes look ready-to-merge now!

**Pull request merged**